docs/spec/spec.md

```diff
@@ -1,13 +1,45 @@
 Lets do https://github.com/knative/serving/blob/master/docs/spec/spec.md style.
 
-For:
+# Knative Source API Spec
 
- - JobSource
+## Resource YAML Definitions
 
-the params it takes.
-  what they mean.
+### JobSource
 
-- spec.template
-- spec.backoffLimit
-- spec.parallel
-- etc that are for k8s jobs...
\ No newline at end of file
+```yaml
+apiVersion: sources.knative.dev/v1alpha1
+kind: JobSource
+metadata:
+  name: my-jobsource
+  labels:
+    controller-tools.k8s.io: "1.0"
+spec:
+  # A template is required.
+  template:
+    spec:
+      # is a singleton []corev1.Container.
+      containers: ...
+
+  # Any spec fields valid for a Kubernetes Job are also valid for a JobSource.
+  # The semantics are unchanged as these fields will be passed directly to the
+  # underlying Job.
+  backoffLimit: 2
+
+  # is a corev1.ObjectReference.
+  sink: ...
```

Comment:
it is helpful to have a working example, so perhaps add a ref to some k8s service?